### PR TITLE
Updated documentation for forms V8 - Working with Record data

### LINKED ISSUE #1765: Documentation for working with submitted data In Umbraco Forms 8
## What type of issue is it? (Choose one - delete the others)
Missing documentation

## What article/section is this about? 
https://our.umbraco.com/Documentation/Add-ons/UmbracoForms/Developer/Working-With-Data/

## Describe the issue:
In Umbraco Forms 7 it was possible to get the submitted data from a form and showing it on a page or so. This option is not available in Umbraco Forms 8 or the documentation for this is missing

### PR DESCRIPTION
**Pull request by bergmania**
Updated documentation for https://github.com/umbraco/Umbraco.Forms.Issues/issues/178

fixes #1765

### CODE REVIEW

**Comment by bergmania** [comment]
Please note that the examples can't be tested before Umbraco Forms 8.2 is released

**Comment by sofietoft** [comment]
🎉 

This will close #1765

File: Add-ons/UmbracoForms/Developer/Working-With-Data/index.md [hunk 1]
```diff
@@ -1,106 +1,113 @@
...
-			else{
-				<strong>
-				<a href="@record.Website" target="_blank">@record.Name</a>
-				</strong>
-			}
-			<span>said</span>
-			<p>@record.Comment</p>
-		</li>
-	}
+   @foreach (dynamic record in recordReaderService.GetApprovedRecordsFromPage(Model.Id, 0, 10))
```
Comment (Shazwazza) [inline Add-ons/UmbracoForms/Developer/Working-With-Data/index.md]: I don't think we should be using `dynamic` here... or anywhere?  This method also returns `PagedResult<IRecord>` now so the enumeration and usage of this method will need to be different

Comment (bergmania) [reply Add-ons/UmbracoForms/Developer/Working-With-Data/index.md]: Sure.. Thats a bug.. `dynamic` should be `var`

**Pull request merged**